backend/src/main/java/com/bakdata/conquery/models/datasets/Import.java

```diff
@@ -49,7 +48,9 @@ public Import(int entityBucketSize) {
 	}
 
 	@Valid @NotNull
-	private TableId table;
+	@NsIdRef
```

Comment:
Geht das? In `createForPreprocessing` ziehst du dir doch einfach eine Table aus dem Hut die eigentlich gar nicht existiert.